core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java

```diff
@@ -560,6 +560,15 @@ public boolean supportsFlush() {
   }
 
   /**
+   * This is only used in the test.
```

Comment:
If we're going to do things this way, we might as well use the `@VisibleForTesting` annotation. However, I'd prefer to modify the integration test not to use private API.